src/components/section/helpers.ts

```diff
@@ -44,3 +37,19 @@ export function hasMissingConcept(question: OHRIFormField) {
     question.type == 'obs' && !question.questionOptions.concept && question.questionOptions.rendering !== 'fixed-value'
   );
 }
+
+/**
+ * Make schema transformations especially for originally AFE schemas to match the RFE schema
+ */
+export function transformSchema(question: OHRIFormField) {
+  switch (question.type) {
+    case 'encounterProvider':
+      question.questionOptions.rendering = 'encounterProvider';
```

Comment:
very minor: @kajambiya apologies for my OCD but why do we have the rendering types following different cases? camelCase and kebab-case?

Reply:
Also this function is not transforming the entire schema so I would suggest you rename it because it can be confusing and people can decide to add logic beyond just changing the field definitions